### PR TITLE
Squareline boards

### PR DESCRIPTION
**Pull request by espzav**
# Checklist for new Board Support package or Component

- [x] Component contains License
- [x] Component contains README.md
- [x] Project [README.md](../README.md) updated
- [x] Component contains idf_component.yml file with `url` field defined
- [x] Component was added to CI [upload job](https://github.com/espressif/esp-bsp/blob/master/.github/workflows/upload_component.yml#L17)
- [x] New files were added to CI build job
- [ ] _Optional:_ Component contains unit tests
- [x] CI passing

# Change description
Added missing boards into SquareLine generator. All boards was tested with SquareLine Studio and tested on a board.

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif PTAL

**Comment by tore-espressif** [comment]
@espzav I tested it on s3-eye and it works great!

File: SquareLine/common/main/main.c [hunk 1]
```diff
@@ -34,8 +34,8 @@ void app_main(void)
     /* Initialize display and LVGL */
     bsp_display_start();
 
-    /* Set default display brightness */
-    bsp_display_brightness_set(APP_DISP_DEFAULT_BRIGHTNESS);
+    /* Turn on display backlight */
+    bsp_display_backlight_on();
```
Comment (tore-espressif) [inline SquareLine/common/main/main.c:38]: Could you please remove the #define on line 14 too?

File: SquareLine/boards/esp32_s2_kaluga_kit/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name":"ESP32-S2-Kaluga Kit",
+    "version":"1.0.0",
+    "mcu":"ESP32S2",
+
+    "screen_width":"240",
+    "screen_height":"320",
```
Comment (tore-espressif) [inline SquareLine/boards/esp32_s2_kaluga_kit/manifest.json:7]:
```suggestion
    "screen_width":"320",
    "screen_height":"240",
```

**Review by tore-espressif — APPROVED** [review]
@espzav 2 nitpicks, otherwise LGTM!

**Comment by espzav** [comment]
@tore-espressif Thank you for the review and thank you for the test on kaluga!

**Pull request merged**